Add Reolink Home Hub scene docs (#38038)
home-assistant/home-assistant.github.io: source/_integrations/reolink.markdown

- Chime visitor ringtone
- Hub alarm ringtone
- Hub visitor ringtone
- Hub scene mode (Off, Disarmed, Home, Away)

**PTZ preset** positions can be set in the Reolink app/windows/web client, the names of the presets will be loaded into Home Assistant at the start of the integration. When adding new preset positions, please restart the Reolink integration.

**Play quick reply messages**/**Auto quick reply messages** can be recorded in the Reolink phone app where a name is also supplied. New or updated quick reply messages will be loaded into Home Assistant at the start of the integration. When adding new quick reply messages, please restart the Reolink integration.

> **Hub scene modes** can be set in the Reolink app/client, the names of the scenes will be loaded into Home Assistant at the start of the integration. When adding new custom scenes, please restart the Reolink integration.

Reviewer: ```suggestion
**Hub scene modes** can be set in the Reolink app/client. The scene names are loaded into Home Assistant at the start of the integration. After adding new custom scenes, restart the Reolink integration.
```
